pkg/ansible/generate_test.go

```diff
@@ -106,11 +108,28 @@ func TestGenerateClusterWideVars(t *testing.T) {
 				"oreg_url",
 			},
 		},
+		{
+			name:           "long clusterID",
+			clusterID:      "012345678901234567890123456789-abcde",
+			clusterSpec:    testClusterSpec(),
+			infraSize:      2,
+			clusterVersion: testClusterVersion(),
+			shouldInclude: []string{
+				"openshift_aws_clusterid: 012345678901234567890123456789-abcde",
+				"openshift_aws_elb_master_external_name: 0123456789-abcde-master-external",
+				"openshift_aws_elb_master_internal_name: 0123456789-abcde-master-internal",
+				"openshift_aws_elb_infra_name: 0123456789-abcde-infra",
+				// TODO: Use these instead when the ansible playbook is updated to support it.
+				// "openshift_aws_elb_master_external_name: 0123456789012345678-abcde-cp-ext",
```

Comment:
Whats the issue with the ansible playbook, and is that in progress or something we need to request or add to get done?

Reply:
There is a pending PR that is approved but waiting for the 3.10 branch.